bin/test.sh

```diff
@@ -54,6 +54,7 @@ echo ">>> run tests"
 
 # Run socorro tests
 "${PYTEST}"
+CLOUD_PROVIDER=GCP "${PYTEST}" -m gcp
```

Comment:
The submitter tests use `queue_helper` and `storage_helper`, so we need to do AWS and GCP tests now for `socorro` tests.